pkg/blobplugin/Dockerfile

```diff
@@ -26,7 +26,8 @@ RUN apt-get update && clean-install ca-certificates pkg-config libfuse-dev cmake
 RUN wget -O /tmp/packages-microsoft-prod.deb https://packages.microsoft.com/config/ubuntu/16.04/packages-microsoft-prod.deb && dpkg -i /tmp/packages-microsoft-prod.deb && apt-get update && apt install blobfuse fuse -y && rm -f /tmp/packages-microsoft-prod.deb
 RUN apt remove wget -y
 # this is a workaround to install nfs-common & nfs-kernel-server and don't quit with error
-RUN apt update && apt install nfs-common nfs-kernel-server -y || true
+# WIP: check if updating the kernel version would fix the below error
+# RUN apt update && apt install nfs-common nfs-kernel-server -y || true
```

Comment:
wait, it's not installing `nfs-common`, that's in correct. I will start a new PR to fix this issue.

Reply:
but without installing `nfs-common` this the tests have passed. That means that its no longer needed right?

Reply:
no, there is no nfs e2e tests now

Reply:
oh! ok. let me create an other PR to revert this change.

Reply:
hoo, I see you have already made that change :). Cool.